cfme/physical/overview.py

```diff
@@ -0,0 +1,33 @@
+from navmazing import NavigateToAttribute
+
+from widgetastic_manageiq import StatusBox
+from cfme.base.login import BaseLoggedInPage
+from cfme.utils.appliance.implementations.ui import CFMENavigateStep, navigator
+from cfme.utils.appliance import Navigatable
+
+
+class PhysicalOverview(Navigatable):
+    pass
+
+
+class PhysicalOverviewView(BaseLoggedInPage):
+    providers = StatusBox('Providers')
```

Comment:
There is a ParametrizedStatusBox available now, if you'd like to use it instead of creating separate widgets - your call.